packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Changed
+
+- **BREAKING** Bump peer dependency on `@metamask/preferences-controller` to `^14.0.0` ([#4374](https://github.com/MetaMask/core/pull/4374))
+
```

Comment:
looks like a few other changes need to be documented here

https://github.com/MetaMask/core/compare/@metamask/assets-controllers@32.0.0...HEAD